### ISSUE TITLE
Emacs window effect freezes when toggle-frame-maximized is set to true

### ISSUE THREAD

**Issue by Zlopez** [Schneegans/Burn-My-Windows#86]
## Describe the Bug
When opening Emacs window and the `toggle-frame-maximized` is set to true the effect will froze and stays on the window permanently. See screenshot bellow.

![Screenshot from 2022-02-02 09-07-44](https://user-images.githubusercontent.com/6943409/152116213-4dbc33b3-4da0-47f1-a300-ac83a9b02200.png)

When removing `toggle-frame-maximized` the behavior stops.

1. Install Emacs
2. Add following lines to ~/.emacs
```
;; Start maximised (cross-platf)
(add-hook 'window-setup-hook 'toggle-frame-maximized t)
```
3. Enable Burn My Windows extension
4. Open Emacs window

## Expected Behavior
The effect will finish as when played on other windows

## System
 - Linux distribution [Fedora 35 Silverblue]
 - Burn-My-Windows version [Burn-My-Windows 10]
 - GNOME Shell version: [41.3]

**Comment by Schneegans** [comment]
Thanks for the report! I am pretty sure that this is same issues as #82, isn't it? I hope to fix this with the next update!

**Comment by Zlopez** [comment]
I'm not sure if this is the same issue. It's not starting as fullscreen, it's just that after opening the window it's resized to maximized state automatically. And if you look closely at the screenshot, the size of the effect is the original size of opened window.

**Comment by Schneegans** [comment]
I think you are right. I can reproduce this with other windows as well: Set the animation time of an effect to a large value, then try maximizing the window while the effect is still running.

**Comment by Zlopez** [comment]
I even found out right now, that if you just want to move a tab out of the browser (it creates it's own browser window) the effect freezes as well. It seems that any geometry change to window why effect is active leads to effect being stuck and not finishing.

**Comment by Schneegans** [comment]
I just pushed a potential fix to the branch `fix/#86`. It would be cool if you could test this on your end! To do this, just follow the steps outlined here: https://github.com/Schneegans/Burn-My-Windows#c-cloning-the-latest-version-with-git

Before `make install`, just execute a `git checkout fix/#86`.

For me, this seems to fix #82 as well, so it was indeed a duplicate :stuck_out_tongue:

**Comment by Zlopez** [comment]
With the #87 the issue is fixed. Even the one with the tab in browser :-)
